internal/eea/eea.go

```diff
@@ -156,11 +168,12 @@ func (e *EEA) aggregate(msg *message.Message) (*message.Message, error) {
 		logger.Info().Msg("executor not ready to process event. Queuing in flush cache.")
 
 		_, err := e.querier.EnqueueFlush(ctx, db.EnqueueFlushParams{
-			Entity:        entities.EntityTypeToDB(inf.Type),
-			RepositoryID:  repoID,
-			ArtifactID:    artifactID,
-			PullRequestID: pullRequestID,
-			ProjectID:     projectID,
+			Entity:           entities.EntityTypeToDB(inf.Type),
```

Comment:
Do you plan to get rid of the type field in a future PR? (I know removing this requires a few steps to avoid breaking backwards compatibility)

Reply:
Right, type won't be needed in the future since the type won't matter. We'd be able to solely rely on the entity instance ID.

This PR merely duplicates info and is not making any logic moves yet.